libs/langgraph/tests/test_prebuilt.py

```diff
@@ -2040,3 +2041,9 @@ def foo(a: str, b: int) -> float:
         return 0.0
 
     assert _get_state_args(foo) == {"a": None, "b": "bar"}
+
+
+def test_inspect_react() -> None:
```

Comment:
@vbarda @dqbd this test passes on main rn so not testing the right thing, do you know how to test the issue?

Reply:
updated!